deepsecurity/tenants.py

```diff
@@ -0,0 +1,117 @@
+import core
+import translation
+import json
+import urllib
+
+class Tenants(core.CoreDict):
+  def __init__(self, manager=None):
+    core.CoreDict.__init__(self)
+    self.manager = manager
+    self.log = self.manager.log if self.manager else None
+
+  def get(self):
+    """
+    Get all of the Tenants from Deep Security
+    """
+
+    rest_call = self.manager._get_request_format(api=self.manager.API_TYPE_REST)
+
+    rest_call['call'] = 'tenants'
+    rest_call['query'] = {}
+    rest_call['query']['sID'] = self.manager._sessions[self.manager.API_TYPE_REST]
+
+    response = self.manager._request(rest_call, auth_required=False)
+
+    return response
+
+class Tenant(core.CoreObject):
+  def __init__(self, manager=None, api_response=None, log_func=None):
+    core.CoreObject.__init__(self)
+    self.manager = manager
+    self.log = self.manager.log if self.manager else None
+    if api_response: self._set_properties(api_response, log_func)
```

Comment:
Coding style question for the maintainers, but I think the newline makes this more clear unless using the full ternary.